### PR TITLE
EVG-16185: Add version control toggle

### PR DESCRIPTION
**Pull request by sophstad**
EVG-16185

### Description 
- Add version control toggle to general settings page

### Screenshots
image

### Testing 
- Update unit tests

### CODE REVIEW

**Review by ablack12 — APPROVED** [review]
Feature-wise looks good to me.

**Review by malikchaya2 — APPROVED** [review]

File: src/pages/projectSettings/tabs/GeneralTab/Fields/MoveRepoModal.tsx [hunk 1]
```diff
@@ -72,6 +74,11 @@ export const MoveRepoModal: React.VFC<ModalProps> = ({
         {repoOwner}/{repoName}. Attach to an existing repo or create a new one
         to which unconfigured settings in this project will default.
       </p>
+      <p>
+        {/* TODO: Replace with LeafyGreen Select when z-index modal bug has been fixed (PD-1677) */}
+        GitHub Organizations available for use as project owners are:{" "}
+        {githubOrgs.join(", ")}
```
Comment (khelif96) [inline src/pages/projectSettings/tabs/GeneralTab/Fields/MoveRepoModal.tsx:80]: joinWithConjugation?  little oxford comma action?

File: src/pages/projectSettings/tabs/GeneralTab/Fields/RepoConfigField.tsx [hunk 2]
```diff
@@ -34,6 +37,9 @@ export const RepoConfigField: Field = ({
   const ownerOrRepoHasChanges =
     formData.owner !== initialOwner || formData.repo !== initialRepo;
 
+  const { data } = useQuery<GetGithubOrgsQuery>(GET_GITHUB_ORGS);
+  const githubOrgs = data?.spruceConfig?.githubOrgs ?? [];
```
Comment (khelif96) [inline src/pages/projectSettings/tabs/GeneralTab/Fields/RepoConfigField.tsx:41]: nit. destructuring?

File: src/pages/projectSettings/tabs/GeneralTab/getFormSchema.tsx [hunk 3]
```diff
@@ -392,3 +405,16 @@ export const getFormSchema = (
     },
   },
 });
+
+const VersionControlEnabledDescription = (
+  <>
+    Enabling Version Control allows{" "}
+    <Link
+      hideExternalIcon
+      href="https://github.com/evergreen-ci/evergreen/wiki/Project-and-Distro-Settings#version-control"
```
Comment (khelif96) [inline src/pages/projectSettings/tabs/GeneralTab/getFormSchema.tsx:414]: nit. Place this url in the externalResources file. 
https://github.com/evergreen-ci/spruce/blob/40db57059ee5ca9a1a92237e31002abcbe18b045/src/constants/externalResources.ts

File: src/pages/projectSettings/tabs/GeneralTab/getFormSchema.tsx [hunk 4]
```diff
@@ -1,3 +1,4 @@
+import { Link } from "@leafygreen-ui/typography";
```
Comment (khelif96) [inline src/pages/projectSettings/tabs/GeneralTab/getFormSchema.tsx:1]: Use the spruce StyledLinks 

https://github.com/evergreen-ci/spruce/blob/cc67877ac01d9f5558213084a5c3e1fc6a1b0a05/src/components/styles/StyledLink.tsx

**Review by khelif96 — APPROVED** [review]
🚀

**Pull request merged**